batch_job/src/work_queue_factory.c

```diff
@@ -954,6 +1030,17 @@ int main(int argc, char *argv[])
 		}
 	}
 
+	if(!project_regex) {
```

Comment:
if(project_regex) {
using_catalog = 1;
else if(master_name ...)
else
help and exit